modules/nf-core/untar_files/main.nf

```diff
@@ -0,0 +1,52 @@
+process UNTAR_FILES {
+    tag "$archive"
+    label 'process_single'
+
+    conda "conda-forge::sed=4.7 bioconda::grep=3.4 conda-forge::tar=1.34"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+        'ubuntu:20.04' }"
+
+    input:
+    tuple val(meta), path(archive)
+
+    output:
+    tuple val(meta), path("${prefix}/**") , emit: files, optional: true
```

Comment:
Why is this optional now?

Reply:
sorry, a relic from before, thanks for the spot